source/backend/api/Services/DocumentSyncService.cs

```diff
@@ -288,8 +291,8 @@ private void AddMayanIdToPimsDocumentTypes(IList<AddDocumentToMayanWithNameRespo
 
         private void SyncDocumentTypeMetadataTypes(SyncModel model, IList<PimsDocumentTyp> pimsDocumentTypes, ref ExternalBatchResult batchResult)
         {
-            Task<ExternalResult<QueryResult<DocumentType>>> documentTypeTask = mayanDocumentRepository.TryGetDocumentTypesAsync(pageSize: 5000);
-            Task<ExternalResult<QueryResult<MetadataType>>> metadataTypesTask = mayanMetadataRepository.TryGetMetadataTypesAsync(pageSize: 5000);
+            Task<ExternalResponse<QueryResponse<Models.Mayan.Document.DocumentTypeModel>>> documentTypeTask = mayanDocumentRepository.TryGetDocumentTypesAsync(pageSize: 5000);
```

Comment:
Is there a reason why DocumentTypeModel is fully qualified but MetadataTypeModel isn't?